docs/docker-env-setup.md

```diff
@@ -0,0 +1,95 @@
+# Nydus Setup for Docker(Moby) Environment
+## Install Docker Nydus Graph Driver for Docker
+1. For older versions of Docker(Moby) lower than v24, please use [Docker Nydus Graph Driver](https://github.com/nydusaccelerator/docker-nydus-graphdriver)
+
+## Install Nydus Snapshotter for Docker(Moby) with Systemd
+1. Docker(Moby) newer than [5c1d6c957b97321c8577e10ddbffe6e01981617a](https://github.com/moby/moby/commit/5c1d6c957b97321c8577e10ddbffe6e01981617a) is needed on your host. The commit is expected to be included in Docker v24. 
+```
+git clone https://github.com/moby/moby
+cd moby
+make binary
+cd bundles/binary
+sudo systemctl stop docker
+sudo systemctl stop containerd
+sudo cp ./* /usr/bin/
+```
+
+2. Download nydus-snapshotter release tarball from [the release page](https://github.com/containerd/nydus-snapshotter/releases). 
+```
+# Get the stable version
+TAG=v0.6.0
+wget https://ghproxy.com/https://github.com/containerd/nydus-snapshotter/releases/download/"$TAG"/nydus-snapshotter-"$TAG"-x86_64.tgz
+tar -xzvf nydus-snapshotter-"$TAG"-x86_64.tgz
+sudo install -D -m 755 nydus-snapshotter/containerd-nydus-grpc /usr/local/bin
+
+wget -O /etc/nydus/nydusd-config.json https://raw.githubusercontent.com/containerd/nydus-snapshotter/"$TAG"/misc/snapshotter/nydusd-config.fusedev.json
```

Comment:
why we need this considering nydus-snapshotter .tar.gz is already downloaded.

Reply:
Because the .tgz file only contains the nydus snapshotter binary.

Reply:
why .tgz files don't contain other configuration files? it seems an issue that needs to be resolved @changweige @imeoer